components/nimbus/ios/Nimbus Sample/Nimbus Sample/Experiments.swift

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import MozillaAppServices
+
+public enum Experiments {
+    public static var shared: NimbusApi {
+        // Nimbus setup
+
+        // This constructs a `URL` representing the Remote Settings endpoint
+        // we can retrieve experiments from.
+        var nimbusEndpointUrl: URL {
+            var components = URLComponents()
+            components.scheme = "https"
+            components.host = "firefox.settings.services.mozilla.com"
```

Comment:
Should this be the live one, or a stage one?

Reply:
Since we don't really have a way to target the sample unless we add it as an app to Experimenter, I'm not sure pointing this at either prod or stage gains us anything at this point.  Perhaps it would be better to point this at "example.com" and just rely on the local experiment file.  I can see where @boek's debug workflow from Firefox-iOS may be useful here, also.